### PR TITLE
dev-python/latexcodec: new package

### PR DESCRIPTION
**Pull request by WPettersson**
Closes: https://bugs.gentoo.org/720456
Signed-off-by: William Pettersson <william@ewpettersson.se>
Package-Manager: Portage-2.3.89, Repoman-2.3.22

### CODE REVIEW

File: dev-python/latexcodec/latexcodec-2.0.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (thesamesam) [inline dev-python/latexcodec/latexcodec-2.0.0.ebuild:4]: Can we do EAPI=7 here? 

Unless there's a good reason (like eclasses), new packages and such should be the latest EAPI.

File: dev-python/latexcodec/latexcodec-2.0.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+PYTHON_COMPAT=( python3_6 python3_7 python3_8 )
```
Comment (thesamesam) [inline dev-python/latexcodec/latexcodec-2.0.0.ebuild:6]: You can use the nice python3_{6,7,8} syntax here, by the way.

File: dev-python/latexcodec/metadata.xml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <longdescription>
```
Comment (thesamesam) [inline dev-python/latexcodec/metadata.xml:4]: I usually prefer maintainers first but it looks like this is legal.

File: dev-python/latexcodec/latexcodec-2.0.0.ebuild (hunk 1 above)
Comment (WPettersson) [reply dev-python/latexcodec/latexcodec-2.0.0.ebuild]: Updated to EAPI=7.

File: dev-python/latexcodec/latexcodec-2.0.0.ebuild (hunk 2 above)
Comment (WPettersson) [reply dev-python/latexcodec/latexcodec-2.0.0.ebuild]: Switched to nicer syntax :)

File: dev-python/latexcodec/metadata.xml (hunk 3 above)
Comment (WPettersson) [reply dev-python/latexcodec/metadata.xml]: Put maintainers first

**Comment by WPettersson** [comment]
Force-pushed a new commit that addresses issues, and rebases to newer master.

**Comment by WPettersson** [comment]
Force-pushed a commit which rebases onto a more current master.

**Comment by WPettersson** [comment]
Force-pushed a commit which rebases onto a more current master.

**Comment by WPettersson** [comment]
Closed via #16051

**Pull request closed without merging**